types/animejs/index.d.ts

```diff
@@ -0,0 +1,141 @@
+// Type definitions for animejs 2.0
+// Project: http://animejs.com
+// Definitions by: Andrew Babin <https://github.com/A-Babin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type FunctionBasedParamter = (element : HTMLElement, index : number, length : number) => number;
+type AnimeCallbackFunction = (anim : anime.AnimeInstance) => void;
+type AnimeTarget = string | object | HTMLElement | SVGElement | NodeList | null; //Allowing null is necessary because DOM querys may not return anything. 
+
+declare namespace anime {
+	const enum EasingsEnum {

```

Comment:
do not use string enums in declaration files. they will break under `--isolatedModule`.